### ISSUE TITLE
Version mismatches in t/000prereq.t

### ISSUE THREAD

**Issue by eserte** [pmqs/Archive-Zip-SimpleZip#11]
t/000prereq.t fails on some of my smokers like this:
```
#     Failed test (t/000prereq.t at line 54)
#          got: '2.092'
#     expected: '2.091'
# IO::Compress::Lzma version is 2.092, need 2.091

#     Failed test (t/000prereq.t at line 54)
#          got: '2.092'
#     expected: '2.091'
# IO::Uncompress::UnLzma version is 2.092, need 2.091
# 
# 
# perl                      5.026002
# Archive::Zip::SimpleZip   0.033
# Archive::Zip::SimpleUnzip 0.033
# IO::Compress::Base        2.092
# IO::Compress::Zip         2.092
# IO::Compress::Lzma        2.092
# IO::Uncompress::Base      2.092
# IO::Uncompress::Unzip     2.092
# IO::Uncompress::UnLzma    2.092
# Compress::Raw::Zlib       2.092
# Compress::Raw::Bzip2      2.092
# Compress::Raw::Lzma       2.092
# zlib                      1.2.11
# bzip2                     1.0.8, 13-Jul-2019
# lzma                      5.2.4
# 
# 
# Looks like you failed 2 tests of 6.
t/000prereq.t ............. 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/6 subtests 
```

**Comment by pmqs** [comment]
Thanks. Seen that in some of the smokes.

**Comment by pmqs** [comment]
Fixed with 330dfb7dda1b3016eec52dc1c90ff30110aa8f6a

Available in 0.034
